sqlglot/dialects/dialect.py

```diff
@@ -545,6 +545,12 @@ def rename_func(name: str) -> t.Callable[[Generator, exp.Expression], str]:
     return lambda self, expression: self.func(name, *flatten(expression.args.values()))
 
 
+def wrap_func_by_func(
```

Comment:
can we delete this and just do it by hand?

Reply:
Sure